basecore_ext_dep.yaml

```diff
@@ -9,6 +9,6 @@
   "name": "MU_BASECORE",
   "var_name": "BASECORE_PATH",
   "source": "https://github.com/microsoft/mu_basecore.git",
-  "version": "661901f38f33606cc6cdbce7defccfc8b80a84af", # release/202202
+  "version": "0d04a5e280c55eb742deaabcfa97da85a0afe7b2", # release/202208
```

Comment:
Basecore has more recent commits than this one.

Reply:
Thanks, updated.